index-test.js

```diff
@@ -748,4 +748,16 @@ describe('reactElementToJSXString(ReactElement)', () => {
   </div>
 </div>`);
   });
+
+  it('should return the functionValue value when "showFunctions" is true and functionValue is a valid string', () => {
+    expect(
+      reactElementToJSXString(<div fn={() => 'value'}/>, {showFunctions: true, functionValue: 'Replace me'})
```

Comment:
when your component is used like this:

```jsx
reactElementToJSXString(<Component prop={i => i++} />, {functionValue: code => console.log(code)})

// outputs 'i => i++'
```

You can get this with fn.toString();

Example with node repl:

![image](https://cloud.githubusercontent.com/assets/123822/24271957/7e0ec724-101b-11e7-8c63-a1fc1c6725be.png)

If this is too obscure, let me know and we ditch this.